### DISCUSSION TITLE
How to avoid artefacts in color calibrated image?

### DISCUSSION THREAD

**Discussion by camilo1704** [colour-science/colour#1095] [Q&A]
### Question

I have used the example for color calibration using a color checker and the output image seems to have some artifacts. Any idea how to avoid them? Here are the original an the color calibrated images: 
![r](https://user-images.githubusercontent.com/38401855/213528301-66c77f4d-8625-4490-a476-d9eadb50069b.png)
![after_ex](https://user-images.githubusercontent.com/38401855/213528312-5aa5c488-1d3c-4642-9628-feb584185094.png)
Thanks in advance!

**Comment by thomasmansencal** [comment]
Hello @camilo1704,

The image is visibly a highly compressed jpg so it is not entirely surprising. Running a Laplacian filter on it really does show blocking artefacts:

![image](https://user-images.githubusercontent.com/99779/213620919-4d08d81d-af5f-47c4-a667-b93829ff75f0.png)

Without trying to do a complex colour correction, a simple white-balance on the Neutral 5D swatches highlights that this is not going to work:

![image](https://user-images.githubusercontent.com/99779/213621286-3733ca79-ac27-41dd-96eb-be20b4540625.png)

I would recommend sourcing a higher quality uncompressed image.

Cheers,

Thomas
